app/src/main/res/layout/dialog_feedback.xml

```diff
@@ -1,48 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:orientation="vertical"
-  android:paddingLeft="20dp"
-  android:paddingRight="20dp">
-
-  <org.gdg.frisbee.android.widget.FloatLabelLayout
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    app:floatLabelTextAppearance="@style/TextAppearance.GDG.FloatLabel"
-    app:floatLabelPaddingTop="8dp"
-    app:floatLabelPaddingLeft="4dp"
-    app:floatLabelPaddingBottom="0dp">
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="20dp"
+    android:paddingRight="20dp">
 
-    <EditText
-      android:id="@+id/feedback_message_text"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:hint="@string/feedback_message_hint"
-      android:gravity="top|left"
-      android:inputType="textMultiLine|textCapSentences"
-      android:maxLines="4"
-      android:minLines="1"/>
+    <android.support.design.widget.TextInputLayout
+        android:id="@+id/feedback_message_text_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
 
-  </org.gdg.frisbee.android.widget.FloatLabelLayout>
+        <EditText
+            android:id="@+id/feedback_message_text"
```

Comment:
In Android Studio project properties there is an option to specify the tab size and indent. You can set it only for this project. Then CTRL+ALT+L will format the XML code to 2 space indentation. 

When you have a 2space indented file opened, it also suggests you to change the setting.

![screen shot 2015-07-19 at 14 54 46](https://cloud.githubusercontent.com/assets/763339/8765772/30f2016e-2e26-11e5-9ff0-83cae1aae54d.png)